PhpAmqpLib/Helper/Protocol/MethodMap080.php

```diff
@@ -116,7 +116,7 @@ public function get_method($method_sig)
 
     /**
      * @var string $method_sig
-     * @return bool
+     * @return boolean
```

Comment:
You don't want to change the generated classes manually.  These are generated from the `spec/parser.php`.  Also, I think we want to keep `bool`, not `boolean`.